corehq/form_processor/models.py

```diff
@@ -39,5 +43,49 @@ def get(cls, id):
     def form_id(self):
         return self.form_uuid
 
+    def get_xml_element(self):
+        xml = self._get_xml()
+        if not xml:
+            return None
+
+        def _to_xml_element(payload):
+            if isinstance(payload, unicode):
+                payload = payload.encode('utf-8', errors='replace')
+            return etree.fromstring(payload)
+        return _to_xml_element(xml)
+
+    @property
+    def form_data(self):
+        from .utils import convert_xform_to_json
+        xml = self._get_xml()
+        return convert_xform_to_json(xml)
+
+    def _get_xml(self):
+        xform_attachment = self.xformattachmentsql_set.filter(name='form.xml').first()
+        return xform_attachment.read_content()
+
+
+class XFormAttachmentSQL(models.Model):
```

Comment:
I was thinking that it could be useful to have a custom Django JSON field that takes converts the JSON into a jsonobject class:
```python
class Attachment(JsonObject):
    name = StringProperty()

class MyModel(models.Model):
    attachments = JsonObjectField(Attachment)
```

If you think that would be useful I can take a crack at making it
```